### PR TITLE
[WIP] STIG Updates

### PR DESCRIPTION
**Pull request by tbrunell**

### CODE REVIEW

**Comment by shawndwells** [comment]
jenkins add to whitelist

File: shared/xccdf/system/software/gnome.xml [hunk 1]
```diff
@@ -413,21 +413,21 @@ of the information system but does not want to logout because of the temporary n
 <title>Set GNOME3 Screensaver Lock Delay After Activation Period</title>
 <description>
 To activate the locking delay of the screensaver in the GNOME3 desktop when 
-the screensaver is activated, add or set <tt>lock-delay</tt> to <tt>uint32 0</tt> in
+the screensaver is activated, add or set <tt>lock-delay</tt> to <tt>uint32 5</tt> in
```
Comment (redhatrises) [inline shared/xccdf/system/software/gnome.xml]: @shawndwells we should make these integer variables to handle different profile requirements.

Comment (shawndwells) [inline shared/xccdf/system/software/gnome.xml]: @redhatrises totally agree. baby steps as @tbrunell gets up to speed on ssg/coding/scap :)

**Comment by shawndwells** [comment]
Going to go ahead and merge this to get the correct setting in. We'll want to change to variable in the future as @redhatrises called out.

**Pull request merged**